src/ui/AccessibleLinksFormatter.cpp

```diff
@@ -73,6 +74,16 @@ void AccessibleLinksFormatter::Populate(ShuttleGui& S) const
       return;
    }
 
+#ifdef __WXGTK__
+   // Non empty label is required, as wxHyperlinkCtrl would assert otherwise
+   std::unique_ptr<wxHyperlinkCtrl> tempHyperlink
+       = std::make_unique<wxHyperlinkCtrl>(S.GetParent(), wxID_ANY, wxT("temp"), wxString());
+
+   const wxColour hyperlinkColour = tempHyperlink->GetNormalColour();
+
+   tempHyperlink.reset();
+#endif
+
```

Comment:
Looks ok

I would write it this way instead, keeping my function scope clean and avoiding the stack allocation, but that's me :-)

```
const auto hyperlinkColour = [&]{
   return wxHyperlinkCtrl{S.GetParent(), wxID_ANY, L"temp", {}}.GetNormalColour();
}();
```